server/src/testFixtures/java/io/crate/testing/SQLTransportExecutor.java

```diff
@@ -175,6 +210,10 @@ private SQLResponse executeTransportOrJdbc(TestExecutionConfig config,
             LOGGER.trace("Executing with enable_hashjoin=false: {}", stmt);
         }
 
+        if (config.isRuleRandomizationEnabled()) {
+            sessionList.addAll(buildRandomizedRuleSessionSettings(random, random.nextDouble(0.1, 1), LoadedRules.RULES, config.rulesToKeep()));
```

Comment:
Why do you hardcode the factor here? should be passed from the `value()` of the annotation?

Reply:
sure, thx, I was confused.